hip-config.cmake.in

```diff
@@ -159,8 +159,27 @@ if(HIP_COMPILER STREQUAL "clang")
   if(NOT WIN32)
     find_dependency(AMDDeviceLibs)
   endif()
-  set(AMDGPU_TARGETS "gfx900;gfx906;gfx908;gfx90a;gfx1030" CACHE STRING "AMD GPU targets to compile for")
-  set(GPU_TARGETS "${AMDGPU_TARGETS}" CACHE STRING "GPU targets to compile for")
+  set(GPU_DEFAULT_TARGETS "gfx900;gfx906;gfx908")
+  set(GPU_SUPPORTED_TARGETS "gfx701;gfx801;gfx802;gfx803;gfx900;gfx906;gfx908;gfx1010;gfx1011;gfx1012;gfx1030;gfx1031")
```

Comment:
This is not the complete list of supported targets as this is missing the [target features](https://llvm.org/docs/AMDGPUUsage.html#target-features](https://llvm.org/docs/AMDGPUUsage.html#target-features) which is part of the target name as well. Furthermore, iterating over all the target features and default feature is a pretty long list. I think it would be better to not have the variable at all.

Reply:
I will remove `GPU_SUPPORTED_TARGETS` but it helps beginners to select an architecture even if the feature will not be directly chooseable.

Reply:
I removed `GPU_SUPPORTED_TARGETS`